app/services/atmosphere/appliance_vms_manager.rb

```diff
@@ -81,14 +81,14 @@ def instantiate_vm(tmpl, tenant, flavor, name)
       else
         unsatisfied('Unable to assign VM to appliance - please contact administrator')
         Raven.capture_message('Unable to assign VM to appliance',
-            logger: 'error',
-            extra: {
-              is_at_site: server_id,
-              appliance_name: appliance.name,
-              appliance_id: appliance.id,
-              errors: vm.errors.to_json
-            }
-          )
+                              logger: 'error',
+                              extra: {
+                                  is_at_site: server_id,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.